doc/source/barcelona.md

```diff
@@ -0,0 +1,58 @@
+
+## History of name Barcelona
+
+The name Barcelona comes from the ancient Iberian Baŕkeno, attested in an ancient coin inscription found on the right side of the coin in Iberian script as Barkeno in Levantine Iberian script, in ancient Greek sources as Βαρκινών, Barkinṓn; and in Latin as Barcino, Barcilonum and Barcenona.
+
+![Barcelona history](_static/images/barcelona/history.jpg)
+
+Some older sources suggest that the city may have been named after the Carthaginian general Hamilcar Barca, who was supposed to have founded the city in the 3rd century BC,but there is no evidence that Barcelona was ever a Carthaginian settlement, or that its name in antiquity, Barcino, had any connection with the Barcid family of Hamilcar. During the Middle Ages, the city was variously known as Barchinona, Barçalona, Barchelonaa, and Barchenona.
+
+## Architecture 
+
+The city of Barcelona is one of the best-preserved medieval cities in Europe. As a result, visitors will see several aspects of the city’s historical nature that have helped it maintain its structures and essential aspects. The street plan is still recognizably medieval, despite several alterations over time and modernization during the 20th century.
+The historic district contains many monuments, including Gothic churches and winding narrow streets with tiled roofs surrounded by walls at regular intervals.
+
+![Barcelona architecture](_static/images/barcelona/arct.jpg)
+
+The ancient walls were built in the 12th and 13th centuries to defend the populace from attacks by foreign armies. Other famous buildings include the most extensive library in Europe, Casey Mila, Barcelona’s Cathedral, and Park Güell – one of Gaudí’s masterpieces, called an architectonic masterpiece.
+
+## FC Barcelona (Barça)
+
+FC Barcelona is a Spanish football club. The club enjoys massive support throughout the world, having won many trophies. FC Barcelona has had many top players, including Pep Guardiola and Lionel Messi.
+
+![Leo Messi](_static/images/barcelona/messi.jpg)
+
+FC Barcelona certainly has its unique characteristics, as demonstrated by its successful team and the individuals who have played for it. In fact, FC Barcelona is one of the most valuable sports franchises in Europe. Additionally, the Camp Nou stadium and museum rank among Barcelona’s top 10 tourist attractions.
+
+## Location and Climate
+
+The city of Barcelona sits on the Mediterranean coast, surrounded by mountains that form the province of Catalonia. 
+
+![Mediterranean coast](_static/images/barcelona/climate.jpg)
+
+The Mediterranean’s climate, with warm sunny days and cool nights in winter, explains the abundant greenery throughout most of the year. The climate is cooler at night from December to March, but even then, it rarely snows.
+Because of the city’s moderate temperature, residents spend a lot of time outside, from dining al fresco at outdoor restaurants and pubs to hiking or biking in the neighboring mountains. 
```

Comment:
```suggestion
Because of the city’s moderate temperature, residents spend a lot of time outside, from dining al fresco at outdoor restaurants and pubs to hiking or biking in the neighboring mountains.
```